pennylane/_queuing.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the :class:`QueuingContext` abstract base class.
+"""
+import abc
+from collections import OrderedDict
+
+import pennylane as qml
+
+
+class QueuingContext(abc.ABC):
```

Comment:
Method renamed, but otherwise unchanged

Reply:
It recognized the rename, but I also changed the file a bit, so it looks like it just shows everything as a change by default?